llvm/lib/Target/DirectX/DXILDataScalarization.h

```diff
@@ -0,0 +1,35 @@
+//===- DXILDataScalarization.h - Prepare LLVM Module for DXIL Data
+// Legalization----===//
```

Comment:
clang-format does funny things to these comments if they're too long. It's also nice to put the `C++` marker in the banner for editors that look at that sort of thing. Easy enough if we're a little briefer in the description:
```suggestion
//===- DXILDataScalarization.h - Perform DXIL Data Legalization -*- C++ -*-===//
```